ntbs-service/Models/Entities/Notification.cs

```diff
@@ -88,6 +88,8 @@ public Notification()
 
         #endregion
         
+        public bool HasBeenNotified => NotificationStatus == NotificationStatus.Notified || NotificationStatus == NotificationStatus.Legacy;
```

Comment:
(I'm only moving this method form the other `partial` as it didn't feel that it related to Display per se, so this is auxillary, but..) yes, I believe the we are treating all legacy notifications as notified, since we won't be importing drafts. either way this is a bit of a misleading method, as I don't believe "Legacy" is a valid state for a notification in NTBS, only for a banner holding non-imported data. but I don't feel this is the right place to tackle this incongruity :)